rfcs/XXXX-gene-sets/text.md

```diff
@@ -0,0 +1,100 @@
+# Gene Sets Data Format
+
+## [Optional] Subtitle of my design 
+
+**Authors:** [Madison Dunitz](mailto:madison.dunitz@chanzuckerberg.com)
+
+**Approvers:** [Friendly Tech Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly PM Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly Tech Person2](mailto:some.nice.person@chanzuckerberg.com), [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com), [Sara Gerber](mailto:sara.gerber@chanzuckerberg.com) 
+
+## tl;dr 
+
+This RFC contains the details about the data format for the gene set feature. It will support the export of differential
```

Comment:
```suggestion
This RFC contains the details about the data format for the gene set feature. It will support the import and export of differential
```